Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -278,6 +323,8 @@ public struct TLSConfiguration {
                                  keyLogCallback: NIOSSLKeyLogCallback? = nil,
                                  renegotiationSupport: NIORenegotiationSupport) -> TLSConfiguration {
         return TLSConfiguration(cipherSuites: cipherSuites,
+                                verifySignatureAlgorithms: verifySignatureAlgorithms,
+                                signingSignatureAlgorithms: signingSignatureAlgorithms,
```

Comment:
So I know this is very annoying, but adding parameters to functions is a semver major change, even when those parameters have default values. We need to define new functions that take these extra arguments, sadly.